README.md

```diff
@@ -3,26 +3,12 @@
 [![Star on GitHub][github-star-badge]][github-star]
 [![Tweet][twitter-badge]][twitter]
 
+See the Slack clone in action here: https://pusher.github.io/react-slack-clone
```

Comment:
I usually put these in blockquotes to make them stand out like:

> See the Slack clone in action here: https://pusher.github.io/react-slack-clone

Reply:
I don't mind either way :)